Phonebook.Frontend/src/app/shared/states/Table.state.ts

```diff
@@ -74,3 +83,52 @@ export class TableState {
     ctx.patchState({ resultCount: action.resultCount });
   }
 }
+
+@State<SmallTableStateModel>({
+  name: 'smalltablestate',
+  defaults: {
+    visibleColumns: getColumnsAsStringArray(ColumnDefinitions.getSmall()),
```

Comment:
please don't duplicate the whole state. Just add a property to the `TableStateModel`

Reply:
Don't I have to have two states in order to have two different places in local storage?

Reply:
Well the whole state is saved to local storage, either when adding a property to it or when creating a new one.

Reply:
![2020-04-27_10h48_37](https://user-images.githubusercontent.com/61278471/80352795-b73c7d00-8874-11ea-83a3-0f42efb2b89b.png)
 Are you saying that I should add another default property to the state like this? Then it would still be in the same local storage, though, right?

Reply:
yep, its always the same local storage. Let's have a chat about this. I am available in the afternoon.